hack/internal/tools/go.mod

```
@@ -1,6 +1,8 @@
 module sigs.k8s.io/kueue/internal/tools
 
-go 1.23
+go 1.23.0
+
+toolchain go1.23.2
```

```suggestion
go 1.23.0

```

Couldn't we avoid this toolchain version?

---

+1

---

It was automatically added by `go mod tidy` command.

---

What about other go.mod? Did you run there go mod tidy but it didn't add it?

---

Yes.

---

It is okay to add the toolchain, as some dependencies require this version.

---

Makes sense, I'm ok to keep it here then.